theforeman.org/pipelines/test/testKatello.groovy

```diff
@@ -1,175 +1,187 @@
 def katello_versions = [
     'master': [
         'foreman': 'develop',
-        'ruby': '2.5',
+        'ruby': ['2.5', '2.7']
     ],
     'KATELLO-4.0': [
         'foreman': '2.4-stable',
-        'ruby': '2.5'
+        'ruby': ['2.5']
     ],
     'KATELLO-3.18': [
         'foreman': '2.3-stable',
-        'ruby': '2.5'
+        'ruby': ['2.5']
     ],
     'KATELLO-3.17': [
         'foreman': '2.2-stable',
-        'ruby': '2.5'
+        'ruby': ['2.5']
     ],
 ]
 
-def ruby = katello_versions[ghprbTargetBranch]['ruby']
-def foreman_branch = katello_versions[ghprbTargetBranch]['foreman']
+//def ruby_versions = katello_versions[ghprbTargetBranch]['ruby']
+//def foreman_branch = katello_versions[ghprbTargetBranch]['foreman']
+def ruby_versions = katello_versions['master']['ruby']
+def foreman_branch = katello_versions['master']['foreman']
```

Comment:
this doesn't look right? ;)

Reply:
Truth, I am still testing this. Shoulda gone for draft mode since Katello has a failing test.